src/scripts/main.js

```diff
@@ -1,3 +1,297 @@
+        : reverse
+          ? +y - +x
+          : +x - +y;
+    });
+
+  targetType = mouseEvent.target.textContent;
+
+  tableBody.append(...newTableBody);
+};
+
+const addClassToTargetRow = (mouseEvent) => {
+  const targetRow = mouseEvent.target.closest('tr');
+
+  if (mouseEvent.target.closest('thead')) {
+    return;
+  }
+
+  if (!mouseEvent.target.closest('tbody')) {
+    for (const row of rows) {
+      row.classList.remove('active');
+    }
+
+    return;
+  }
+
+  for (const row of rows) {
+    row.classList.remove('active');
+  }
+
+  targetRow.classList.add('active');
+};
+
+tableHeader.addEventListener('click', sortTableBody);
+document.body.addEventListener('click', addClassToTargetRow);
+
+document.body.insertAdjacentHTML('beforeend', `
+    <form class="new-employee-form">
+      <label>
+        Name:
+
+        <input
+          name="name"
+          type="text"
+          data-qa="name"
+          required
+        >
+      </label>
+
+      <label>
+        Position:
+
+        <input
+        name="name"
+        type="text"
+        data-qa="position"
+        required
+        >
+      </label>
+
+      <label>
+        Office:
+
+        <select
+          name="office"
+          data-qa="office"
+          required
+        >
+          <option value="Tokyo">Tokyo</option>
+          <option value="Singapore">Singapore</option>
+          <option value="London">London</option>
+          <option value="New York">New York</option>
+          <option value="Edinburgh">Edinburgh</option>
+          <option value="San Francisco">San Francisco</option>
+        </select>
+      </label>
+
+      <label>
+        Age:
+
+        <input
+          name="age"
+          type="number"
+          data-qa="age"
+          required
+        >
+      </label>
+
+      <label>
+        Salary:
+
+        <input
+          name="salary"
+          type="number"
+          data-qa="salary"
+          required
+        >
+      </label>
+
+      <button>Save to table</button>
+    </form>
+`);
+
+document.body.style = 'align-items: start;';
+
+const form = document.querySelector('.new-employee-form');
+const personNameLength = 4;
+const minAge = 18;
+const maxAge = 90;
+
+const addNewPerson = (formEvent) => {
+  formEvent.preventDefault();
+
+  const namePerson = document.querySelector("input[data-qa='name']").value;
+  const position = document.querySelector("input[data-qa='position']").value;
+  const office = document.querySelector("select[data-qa='office']").value;
+  const age = +document.querySelector("input[data-qa='age']").value;
+  const salary = +document.querySelector("input[data-qa='salary']").value;
+
+  if (namePerson.length < personNameLength) {
+    pushNotification(
+      'Sorry (-_-)',
+      'The name must be longer than 4 letters',
+      'error',
+    );
+
+    return;
+  }
+
+  if (age < minAge || age > maxAge) {
+    pushNotification(
+      'Sorry (-_-)',
+      'age must be more than 18 and less than 90',
```

Comment:
`'age must be more than 18 and less than 90'` repeated twice in the code: move this to constant.